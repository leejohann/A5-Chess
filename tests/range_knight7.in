move 0 6 0 5
move 6 0 5 2
quit
