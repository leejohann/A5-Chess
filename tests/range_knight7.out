RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKB R
PPPPPPPP
 _ _ N _
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn

